docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -1014,6 +1060,57 @@ POST api/detection_engine/rules
 }
 --------------------------------------------------
 
+
+*Example 8*
+
+Query rule that searches for processes started by MS Office and suppress alerts by `process.parent.name` field on 5 hours time period:
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules
+{
+  "rule_id": "process_started_by_ms_office_program",
+  "risk_score": 50,
+  "description": "Process started by MS Office program - possible payload",
+  "interval": "1h", <1>
+  "name": "MS Office child process",
+  "severity": "low",
+  "tags": [
+   "child process",
+   "ms office"
+   ],
+  "type": "query",
+  "from": "now-70m", <2>
```

Comment:
I overlooked this earlier -- we can remove the callout syntax here, because we don't need to highlight these fields in this context (looks like this was copied from Example 1 above).
```suggestion
  "interval": "1h",
  "name": "MS Office child process",
  "severity": "low",
  "tags": [
   "child process",
   "ms office"
   ],
  "type": "query",
  "from": "now-70m",
```